src/pytypes/onefuzztypes/models.py

```diff
@@ -557,13 +557,37 @@ class NodeHeartbeatEntry(BaseModel):
     data: List[Dict[str, HeartbeatType]]
 
 
+class StopNodeCommand(BaseModel):
```

Comment:
Note, these are just moved from later in models.py to enable us to not have to call `Node.update_forward_refs()`